config/config.go

```diff
@@ -172,6 +172,7 @@ func ParseConfig(configPath string) (Config, error) {
 	}
 
 	viper.AutomaticEnv()
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
```

Comment:
Can you add a comment here for how this fixes nested environment vars

Reply:
https://github.com/ojo-network/price-feeder/pull/40/commits/03e4af9baa193d031bce8bff6c2642d24d6e28f9